modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb

```diff
@@ -9,9 +9,9 @@ class MetasploitModule < Msf::Exploit::Remote
 
   def initialize(info = {})
     super(update_info(info,
-      'Name'            => 'Trixbox CE <= v2.8.0.4 Authenticated Remote Command Execution',
+      'Name'            => 'Trixbox CE v2.8.0.4 Authenticated Remote Command Execution',
```

Comment:
Only this line needed to be changed. However I would change it to `Trixbox CE Authenticated Remote Command Execution`. Better yet, if you can add some specifics on **which component** is affected, I would include this in the title as well.

Reply:
Updated

Reply:
I'd call it something like:

* Trixbox CE v2.8.0.4 endpoint_devicemap.php Authenticated Remote Command Execution

The file name should also be similar to the module name. Suggest something like:

* trixbox_ce_endpoint_devicemap_rce; or
* trixbox_ce_endpoint_devicemap_cmd_exec

Also, I'm guessing `CE` stands for `Community Edition`. Is this vulnerability likely to affect other editions?

Reply:
Hello @bcoles, what about title "**Trixbox CE v2.8.0.4 (and prior versions) endpoint_devicemap.php Authenticated Remote Command Execution**" and filename "**trixbox_ce_endpoint_devicemap_rce**" 

Yes it's also affects previous versions (e.g. [2.6](https://master.dl.sourceforge.net/project/asteriskathome/trixbox%20CE/trixbox%202.6/trixboxCE-livecd-20081006.iso))

Reply:
There is Trixbox Pro which is the commercial version Trixbox.

Reply:
Do you know if Pro shares the same codebase as CE? Is it likely to also be affected?

If so, `CE` can be removed from the module name.

Reply:
I don't know that, i am not able to verify it (that's why i added "CE")